emission/tests/storageTests/token_test.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import logging
+import uuid
+import json
+
+import emission.core.get_database as edb
+
+import emission.storage.decorations.token_queries as esdt
+import emission.net.auth.token_list as tl
+
+
+class TestPlaceQueries(unittest.TestCase):
```

Comment:
the class should match the file name and the test names should follow the existing naming convention